sqlitecluster/SQLite.cpp

```diff
@@ -444,6 +446,7 @@ string SQLite::read(const string& query) {
 
 bool SQLite::read(const string& query, SQResult& result) {
     uint64_t before = STimeNow();
+    // TODO: Support query re-writing.
```

Comment:
Hmmm, why would we need to support rewriting of reads? The caller should be able to re do the read several times and discard the results it doesn't want. I don't think we need to support re-writing these.